templates/base.html.twig

```diff
@@ -73,6 +73,8 @@
         <meta name="robots" content="unavailable_after: {{ page.expiryDate }}"/>
     {% endif %}
     <link rel="home" href="https://www.w3.org/"/>
+    <link rel="shortcut icon" href="https://www.w3.org/assets/logos/w3c/favicon-32.png"/>
+    <link rel="apple-touch-icon" href="https://www.w3.org/assets/logos/w3c/favicon-180.png"/>
```

Comment:
Shortcut seems deprecated as indicated on https://html.spec.whatwg.org/dev/links.html#rel-icon
> For historical reasons, the icon keyword may be preceded by the keyword "shortcut".

Also adding the file type seems expected as well.

```suggestion
    <link rel="icon" type="image/png" href="https://www.w3.org/assets/logos/w3c/favicon-32.png"/>
    <link rel="apple-touch-icon" type="image/png" href="https://www.w3.org/assets/logos/w3c/favicon-180.png"/>
```